.eslintrc.js

```diff
@@ -85,7 +85,7 @@ module.exports = {
 		'@typescript-eslint/consistent-type-definitions': 'warn',
 		'@typescript-eslint/no-empty-function': 'warn',
 		'@typescript-eslint/no-empty-interface': 'warn',
-		'@typescript-eslint/no-floating-promises': 'error',
+		'@typescript-eslint/no-floating-promises': 'warn',
```

Comment:
Was this because of the ` reportWebVitals()` call?

Reply:
It was. Alternatively, we can add a `ts-ignore` to the `reportWebVitals()` call and return this rule to an error, which is probably safer.